src/app/api/services/dot-workflow/dot-workflow.service.ts

```diff
@@ -59,4 +59,21 @@ export class DotWorkflowService {
             })
             .pipe(pluck('entity'));
     }
+
+
+    /**
+     * TBD
+     * @param content data
+     * @returns Content
+     */
+    fireDefaultWorkflowAction(data: any): Observable<any> {
```

Comment:
We need a `dot-action.service`, this is a completely different API/Endpoints.

Reply:
And we should move this: https://github.com/dotCMS/core-web/blob/63de3dcdf60b645b0531dedaca0fcdb05b0c67d3/src/app/api/services/dot-workflow/dot-workflow.service.ts#L54
there.